backend/app.py

```diff
@@ -9,79 +9,49 @@
     isic_data = data['isic']
     wiki_data = data['wiki']
     yandex_data = data['yandex']
+    data = kaggle_data + isic_data + wiki_data + yandex_data
+    shuffle(data)
 
 app = Flask(__name__)
 
 
-def randomize(data, n_images):
-    """Get randomly `n_images` from data.
-
-    Return the unchanged data
-    if the data size is smaller or equal to `n_images`.
-    """
-    if len(data) > n_images:
-        shuffle(data)
-        data = data[:n_images]
-    return data
-
-
-@app.route('/kaggle/<int:n_images>')
-def get_kaggle(n_images):
-    """Get `n_images` images from Kaggle API."""
-    return {'data': randomize(kaggle_data, n_images)}
-
-
-@app.route('/isic/<int:n_images>')
-def get_isic(n_images):
-    """Get `n_images` images from ISIC API."""
-    return {'data': randomize(isic_data, n_images)}
-
+def get_page(data, page):
```

Comment:
Do we have `page = 0`? Or our pages start from 1?

Reply:
We're programmers. We start from 0.

Reply:
Oh. I was confused with simple math.